runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -32,15 +32,16 @@
 import java.awt.image.PixelGrabber;
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.ArrayList;
 import javax.imageio.ImageIO;
 import javax.inject.Inject;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.Client;
-import net.runelite.api.SpriteID;
-import net.runelite.api.SpritePixels;
+import net.runelite.api.*;
```

Comment:
[Don't use wildcard imports for non-static imports.](https://github.com/runelite/runelite/wiki/Code-Conventions#imports)

Reply:
Fixed